### ISSUE TITLE
Exception parsing epub 3.0.1 with roman numeral page numbers

### ISSUE THREAD

**Issue by duncanawoods** [asido/EpubSharp#7]
I am trying to parse an EPUB 3.0.1 which successfully validates using EpubCheck version 4.0.2.

EpubReader.Read()  throws on line 41 of NcxReader.cs.

```
Value = (int?)elem.Attribute(NcxPageTarget.Attributes.Value)
```

I cannot share the book but the content of the element is as follows. As you can see, the value of the attribute is "i" and not an integer.

```
<pageTarget id="navPoint43" type="normal" value="i" xmlns="http://www.daisy.org/z3986/2005/ncx/">
  <navLabel>
    <text>i</text>
  </navLabel>
  <content src="xhtml/02_Title_Page.xhtml#page_i" />
</pageTarget>
```

Stack trace:

```
Unhandled Exception: System.FormatException: Input string was not in a correct format.
   at System.Number.StringToNumber(ReadOnlySpan`1 str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   at System.Number.ParseInt32(ReadOnlySpan`1 s, NumberStyles style, NumberFormatInfo info)
   at System.Int32.Parse(String s, NumberStyles style, IFormatProvider provider)
   at System.Xml.XmlConvert.ToInt32(String s)
   at System.Xml.Linq.XAttribute.op_Explicit(XAttribute attribute)
   at EpubSharp.Format.Readers.NcxReader.<>c.<Read>b__0_1(XElement elem)
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.ToList()
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at EpubSharp.Format.Readers.NcxReader.Read(XDocument xml)
   at EpubSharp.EpubReader.Read(Stream stream, Boolean leaveOpen)
```

The books parses without exceptions if NcxPageTarget.Value is changed to a string.

**Comment by asido** [comment]
Fixed in #9.

**Comment by pedasmith** [comment]
Very cool and thank you! I've updated my code to use your latest library -- pedasmith/VoraciousEBookReader: The source code for the Voracious EBook reader (github.com)
(I haven't made a full store release of my app yet that uses this, though)
FWIW: I have a list of things I learned about EPUB files as I made my program: VoraciousEBookReader/ThingsToKnowAboutEpubFiles.md at main · pedasmith/VoraciousEBookReader (github.com) )

Peter
    On Monday, December 21, 2020, 04:01:11 AM PST, Arvydas Sidorenko ***@***.***> wrote:  
 
 


Closed #7.
